src/components/TradeWidget/index.tsx

```diff
@@ -119,7 +119,8 @@ const validFromId: TradeFormTokenId = 'validFrom'
 const validUntilId: TradeFormTokenId = 'validUntil'
 
 // Grab CONFIG tokens
-const { sellToken: initialSellToken, receiveToken: initialReceiveToken } = CONFIG.initialTokenSelection
+const initialTokenSelection = CONFIG.initialTokenSelection
+const { sellToken: initialSellTokenDefault, receiveToken: initialReceiveTokenDefault } = initialTokenSelection
```

Comment:
why not 1 line this?

Reply:
yes, sure

Reply:
well, how?

Reply:
here was my implementation when i tried this yesterday:

```ts
// Grab CONFIG tokens
const {
  sellToken: sellTokenConfig,
  receiveToken: receiveTokenConfig,
  networks: networksConfig,
} = CONFIG.initialTokenSelection

function getNetworkSpecificDefaultTokensFromURL(
  networkId: number,
): { initialSellToken: string; initialReceiveToken: string } {
  return {
    initialSellToken: networksConfig[networkId]?.sellToken || sellTokenConfig,
    initialReceiveToken: networksConfig[networkId]?.receiveToken || receiveTokenConfig,
  }
}
```

Reply:
given the times, and that u access the `networksConfig` two times. I think I'll just use mine, although, I think doing a util or placing this function I think somewhere central i think is a good thing. Feel free to improve the RC1